tests/bud/buildkit-mount-from/Dockerfilecachefrom

```diff
@@ -0,0 +1,8 @@
+FROM alpine as builder
+RUN mkdir subdir
+COPY hello .
+
+FROM alpine as second
+RUN mkdir /test
+# use another image as cache source
```

Comment:
Suggest "another stage" instead of "another image".

Reply:
fixed